openstack/prometheus-openstack/Chart.yaml

```
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A helm chart for OpenStack monitoring tools.
 name: openstack-prometheus
-version: 0.1.2
+version: 6.2.0
```

This should rather be increased in small steps, as it is chart agnostic. Since we're bumping a major version, we can increment a little further.

```suggestion
version: 1.0.0
```